config/sync/core.entity_form_display.node.author.default.yml

```diff
@@ -3,7 +3,9 @@ langcode: en
 status: true
 dependencies:
   config:
+    - field.field.node.author.field_a_author_content
```

Comment:
You don't need this field you can delete it.

Reply:
done